src/SmartComponents/FormComponents/DescriptiveButton.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Button } from '@patternfly/react-core';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import useFormApi from '@data-driven-forms/react-form-renderer/use-form-api';
+import useFieldApi from '@data-driven-forms/react-form-renderer/use-field-api';
+import './descriptiveCheckbox.scss';
+import {
+  ExclamationTriangleIcon,
+  InfoCircleIcon,
+} from '@patternfly/react-icons';
+
+// eslint-disable-next-line no-unused-vars
+const DescriptiveButton = (props) => {
+  const {
+    group,
+    section,
+    label,
+    beforeOnChange,
+    title,
+    description,
+    isDanger,
+    isGlobal,
+    checkedWarning,
+    infoMessage,
+    input: { onChange, checked, ...input },
+  } = useFieldApi({
+    ...props,
+    type: 'checkbox',
+  });
+  const formOptions = useFormApi();
+
+  return (
+    <Button
+      variant="secondary"
+      {...input}
+      id={`button-${input.name}`}
+      onClick={(event, ...rest) => {
+        beforeOnChange(
+          isGlobal,
+          checked,
+          formOptions,
+          group,
+          section,
+          props.name
+        );
+        onChange(!checked, event, ...rest);
+      }}
+      data-type="descriptive-checkbox"
+      style={{ width: '125px' }}
```

Comment:
use `fit-content` instead. Preferably not directly using `style`, but wrap it in some class